src/plugins/kibana_react/public/split_panel/containers/panel_container.tsx

```diff
@@ -91,6 +91,16 @@ export function PanelsContainer({
     [onPanelWidthChange]
   );
 
+  useEffect(() => {
+    // For now we only support bi-split
+    if (thirdChild) {
+      // eslint-disable-next-line no-console
+      console.warn(
+        '[Split Panels Container] Detected more than two children; ignoring additional children.'
+      );
+    }
+  }, [thirdChild ? 1 : 0]);
```

Comment:
`thirdChild` could be `null` or `undefined` or `""` (empty string), maybe

```suggestion
  }, [children.length > 2]);
```